### PR TITLE
Filter extensions by category

### LINKED ISSUE #77220: code --list-extensions by category
Just like I can do `@installed @category:themes` in the Extensions Search Bar, I want something like `code --list-extensions @category:themes` which will only list installed themes.

It comes handy to copy all the installed themes rather than manually entering them one by one.

Same with other categories would be cool :)

### PR DESCRIPTION
**Pull request by lramos15**
Closes #77220. This allows the user to also provide a `--category` flag with `--list-extensions` to filter the extensions down to just the ones in the category which you have provided.

### CODE REVIEW

File: build/azure-pipelines/common/extract-telemetry.sh [hunk 1]
```diff
@@ -4,7 +4,7 @@ set -e
 cd $BUILD_STAGINGDIRECTORY
 git clone https://github.com/microsoft/vscode-telemetry-extractor.git
 cd vscode-telemetry-extractor
-git checkout 4e64f3de30f8fccb58ebdc0d85c4861a135d46cf
+git checkout f538e3157c84d1bd0b239dfc5ebccac226006d58
```
Comment (lramos15) [inline build/azure-pipelines/common/extract-telemetry.sh]: I don't know why this is saying there's a change, I think my fork is just all messed up, this can safely be ignored as I have another PR that is throwing most of this build script away. This is unrelated to the filter extensions by category.

Comment (sandy081) [reply build/azure-pipelines/common/extract-telemetry.sh]: Ignoring this PR as mentioned.

**Review by sandy081 — APPROVED** [review]
